klarrio-build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dockerRepository": "registry.cp.kpn-dsh.com/opensource/kafka-monitor",
+  "version": {
+  	"upstream": "2.5.10",
+  	"klarrio": "1.0.2"
+  },
+  "buildCommand": "rm -rf build; ./gradlew jar",
```

Comment:
hint: `./gradlew clean jar` achieves the same.

Reply:
aha, good one 👍